utils/bootstrap/helpers/index.mjs

```diff
@@ -17,7 +17,7 @@ export async function checkRequiredProgramsExist(programs) {
 }
 
 export async function copyTemplate(directory) {
-  return await fs.copy('scripts/bootstrap/template', directory);
+  return await fs.copy('utils/bootstrap/template', directory);
```

Comment:
I thought the docs said bootstrap was evil? haha maybe it's useful in some cases?

Reply:
Bootstrap here is just a random name for the folder nothing to do with [Bootstrap](https://getbootstrap.com).
I'm not sure what bootstrap the doc is talking about 😉